src/ResourceManager/AzureBatch/Commands.Batch/Locations/GetBatchLocationQuotasCommand.cs

```diff
@@ -18,8 +18,8 @@
 
 namespace Microsoft.Azure.Commands.Batch
 {
-    [Cmdlet(VerbsCommon.Get, Constants.AzureRmBatchSubscriptionQuotas), OutputType(typeof(PSBatchSubscriptionQuotas))]
-    public class GetBatchSubscriptionQuotasCommand : BatchCmdletBase
+    [Cmdlet(VerbsCommon.Get, Constants.AzureRmBatchLocationQuotas), OutputType(typeof(PSBatchLocationQuotas))]
```

Comment:
@jasper-schneider this is a breaking change; you are changing the name of a cmdlet without adding an alias to the original name. Existing scripts that use the cmdlet `Get-AzureRmBatchSubscriptionQuotas` will no longer work.

Also, the output type has changed, which is another breaking change.